src/applications/personalization/dashboard-2/components/Dashboard.jsx

```diff
@@ -76,7 +76,7 @@ const Dashboard = ({
     return <Loader />;
   } else {
     return (
-      <>
+      <div className="dashboard">
```

Comment:
Added this class to make sure we nest all dashboard specific css under this class.